libs/sdk/e2e-schematics/generators.json

```diff
@@ -4,37 +4,27 @@
   "version": "1.0.0",
   "generators": {
     "component-e2e": {
-      "factory": "./src/generators/component-e2e",
+      "factory": "./src/generators/component-e2e/index",
```

Comment:
These generators stopped working with an error about can't find the module. Adding `/index` fixed the error.